tests/e2e/mod.rs

```diff
@@ -91,11 +93,12 @@ fn run_subgraphs_retail_supergraph(
     // Although the retail supergraph package.json has a `dev:subgraphs` script, windows can't
     // recognize the `NODE_ENV=dev` preprended variable; so, we have to remake that command in a
     // way that windows can understand
-    let mut cmd = Command::new("npx");
-    cmd.env("NODE_ENV", "dev");
-    cmd.args(["nodemon", "index.js"])
-        .current_dir(retail_supergraph.get_working_directory());
-    cmd.spawn().expect("Could not spawn subgraph process");
+    Command::new("npx")
+        .env("NODE_ENV", "dev")
+        .args(["nodemon", "index.js"])
+        .current_dir(retail_supergraph.get_working_directory())
+        .spawn()
+        .expect("Could not spawn subgraph process");
```

Comment:
Do we need to do something with the output/result of this command?

Reply:
So this is a weird one but no, we're basically spinning up this command to run all the retail subgraphs and then we leave it running in the background so other bits of the tests can interact with it. It's further complicated by marking it as `#[once]` within the test framework so it only happens once but then it isn't dropped due to limitations of the test framework. 

Basically it's a bit messy but it does what we want and since it mostly runs in CI that's fine